app/views/products/_bulk_download.html.erb

```diff
@@ -10,7 +10,7 @@
 <% num_measure_tests_ready = product.product_tests.measure_tests.where(state: :ready).count %>
 <% if num_measure_tests_ready == num_measure_tests %>
   <p>This download contains a folder for each measure selected for this product. Inside these folders are XML documents for each patient associated with that measure.</p>
-  <%= form_for product, url: { controller: 'products', action: 'patients' }, :html => { :method => 'GET' } do |f| %>
+  <%= form_for download_full_test_deck_product_path(product), :html => { :method => 'GET' } do |f| %>
```

Comment:
The route is still in there. If there's a more appropriate one to use, we can use that, but we shouldn't be using url: controller/action unless we have a really good reason.